packages/models/src/quickChat.ts

```diff
@@ -0,0 +1,10 @@
+export enum QuickChatOption {
+	GG = "GG",
+	HAPPY = "😃",
+	SHOCKED = "😱",
+	ANGRY = "😠"
+}
+// this can be expanded for phrase to either equal a quickChatOption, or to equal a string provided by the player (non-quick chat);
+export type QuickChatValue = {
```

Comment:
Get rid of this for now, move options to constants file

Reply:
+ change into array, map array to QuickChatButtons array and replace buttons with QuickChatButtonsArray

Reply:
Compromised on keeping enum where it is but mapping buttons array from the enum itself - in future should only have to add quick chat options to this enum, and they will auto populate in quickChatButtonsArray